test/StreamJsonRpc.Tests/JsonRpcProxyGenerationTests.cs

```diff
@@ -137,14 +134,27 @@ public interface IServerWithGenericMethod
         Task AddAsync<T>(T a, T b);
     }
 
-    internal interface IServerInternal : StreamJsonRpc.Tests.ExternalAssembly.ISomeInternalProxyInterface, IServerInternalWithInternalTypesFromOtherAssemblies
+    internal interface IServerInternal : ExAssembly.ISomeInternalProxyInterface, IServerInternalWithInternalTypesFromOtherAssemblies
     {
         Task<int> AddAsync(int a, int b);
     }
 
     internal interface IServerInternalWithInternalTypesFromOtherAssemblies
     {
-        Task<StreamJsonRpc.Tests.ExternalAssembly.SomeOtherInternalType> SomeMethodAsync();
+        Task<ExAssembly.SomeOtherInternalType> SomeMethodAsync();
+    }
+
+    internal interface IRemoteService
+    {
+        internal interface ICallback : ExAssembly.IInternalGenericInterface<ExAssembly.SomeOtherInternalType?>
+        {
+        }
+    }
+
+    [Fact]
+    public void Tomas_Internal()
```

Comment:
Is it possible to update the test name to describe the scenario better?

Reply:
Isn't my name self-explanatory? ;)